test/loadmodels.jl

```diff
@@ -286,7 +295,14 @@ end
         SBML.extensive_kinetic_math(m, m.reactions["reaction1"].kinetic_math).args[1].args[2]
     @test subterm.fn == "/"
     @test subterm.args[1] == SBML.MathIdent("S1")
-    @test isapprox(subterm.args[2].val, 1.0)
+    @test subterm.args[2] == SBML.MathVal(1.0)
+
+    m = readSBML(joinpath(@__DIR__, "data", "sbml00140.xml"))
+
+    subterm = SBML.extensive_kinetic_math(m, m.reactions["reaction1"].kinetic_math).args[2]
+    @test subterm.fn == "/"
+    @test subterm.args[1] == SBML.MathIdent("S1")
+    @test subterm.args[2] == SBML.MathVal(5.0)
```

Comment:
@paulflang Now I'm at this -- the size gets substitued in there if it is available in compartment.size. Is this still desirable? If people are supposed to substitute for compartment variables themselves, I can remove the functionality altogether.